### ISSUE TITLE
Queue signs stay green / selected when you queue another sign

### ISSUE THREAD

**Issue by ProfX33** [mcpvp-club/pvp-club#22]
### Describe the bug

1) Click the queue sign for any gamemode. Then it turns green and changes to "1/2 Queued"
2) Click another queue sign to join it
3) The first sign you have clicked is still green and showing "1/2 Queued", even though you are no longer in that queue.
(In chat it says:
Removed you from the queue.
Successfully added you to the queue!
)

### Expected behavior

1) see above
2) see above
3) The first queue sign you clicked should appear unselected again and show "0/2 Queued"

### To reproduce

https://medal.tv/games/minecraft/clips/iuIfN91C6UBDgM7yg/d1337jQS56gR?invite=cr-MSwxTzgsMjI5NzYyNDYyLA

### Server region

EU (havent checked other regions, but probably those aswell)

### Additional context

I am not sure if this is supposed to be some feature where you can join multiple queues at once, but I have checked the #updates channel on the discord and it doesnt seem to be the case. (Even if it was supposed to be like that it would still be bugged)

### Checklist

- [X] I have checked for any identical bug reports already open.

**Comment by Jyguy** [comment]
You are correct that it is a bug; it is supposed to update the old sign once you click on another one.

This was mentioned in #status on the Discord but I forgot to make an issue about it here. This was introduced by a performance optimization made the other day.
To fix this it will require a bit of rewriting, it'll be fixed when I get to it.
